docs/architecture/adr-014-proportional-slashing.md

```diff
@@ -18,39 +18,31 @@ To solve this problem, we will implement a procedure called Proportional Slashin
 ```
 
-Now, if someone splits a validator of 10% into two validators of 5% each which both fault, then they both fault in the same time frame, they both will still get slashed at the sum 10% amount.
+Now, if someone splits a validator of 10% into two validators of 5% each which both fault, then they both fault in the same time frame, they both will get slashed at the sum 10% amount.
 
-However, an operator might still choose to split up their stake across multiple accounts with hopes that if any of them fault independently, they will not get slashed at the full amount.  In the case that the validators do fault together, they will get slashed the same amount as if they were one entity.  There is no con to splitting up.  However, if operators are going to split up their stake without actually decorrelating their setups, this also causes a negative externality to the network as it fills up validator slots that could have gone to others or increases the commit size.  In order to disincentivize this, we want it to be the case such that splitting up a validator into multiple validators and they fault together is punished more heavily that keeping it as a single validator that faults.
+However in practice, we likely don't want a linear relation between amount of stake at fault, and the percentage of stake to slash. In particular, solely 5% of stake double signing effectively did nothing to threaten security, whereas 30% of stake being at fault clearly merits a slashing factor. A linear relation would require a factor of 6 gap between these two, whereas the difference in risk posed to the network is much larger. We propose using S-curves (formally [logistic functions](https://en.wikipedia.org/wiki/Logistic_function) to solve this). S-Curves capture the desired criterion quite well. They allow the slashing factor to be minimal for small values, and then grow very rapidly near some threshold point where the risk posed becomes notable.
 
-We can achieve this by not only taking into account the sum of the percentages of the validators that faulted, but also the *number* of validators that faulted in the window.  One general form for an equation that fits this desired property looks like this:
-
-```
-slash_amount = k * ((power_1)^(1/r) + (power_2)^(1/r) + ... + (power_n)^(1/r))^r // where k and r are both on-chain constants
-```
+#### Parameterization
 
-So now, for example, assuming k=1 and r=2, if one validator of 10% faults, it gets a 10% slash, while if two validators of 5% each fault together, they both get a 20% slash ((sqrt(0.05)+sqrt(0.05))^2).
+This requires parameterizing a logistic function. It is very well understood how to parameterize this. It has four parameters:
+1) A minimum slashing factor
+2) A maximum slashing factor
+3) The inflection point of the S-curve (essentially where do you want to center the S)
+4) The rate of growth of the S-curve (How elongated is the S)
```

Comment:
can you provide an example for a given logistic function? Which default parameter values would you recommend for the implementation of this feature?

Reply:
I don't have a strong recommendation for default parameters.  In the associated governance proposal, I suggest initial parameterization be done to keep it as close to the current constant slash as possible.  Then we can make a governance proposal to reparameterize.